exporter/templates/cat1/r3/_2.16.840.1.113883.10.20.24.3.54.xml

```diff
@@ -5,26 +5,12 @@
     <templateId root="2.16.840.1.113883.10.20.22.4.79" extension="2014-06-09"/>
     <!-- Patient Characteristic Expired -->
     <templateId root="2.16.840.1.113883.10.20.24.3.54" extension="2014-12-01"/>
-    <id root="1.3.6.1.4.1.115" extension="{{.EntrySection.Entry.ObjectIdentifier.ID}}"/>
+    <id root="1.3.6.1.4.1.115" extension="{{newRandom}}"/>
```

Comment:
Yes, @dczulada pointed out that we are fine using `newRandom` here. I think it's because the expired template can only occur once